src/ir/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(ir_SOURCES
   ExpressionManipulator.cpp
   LocalGraph.cpp
   ReFinalize.cpp
+  stack-utils.cpp
```

Comment:
Surveying the subdirectories of src, we have asmjs, emscripten-optimizer, support, tools, and wasm using (mostly) `kebab-case` (with some `snake_case` thrown in) for their .cpp files and only ir and passes using `PascalCase`. I kind of understand passes using PascalCase because each .cpp is named for the pass class it contains, but I think generally it would be good to standardize on kebab-case.

Reply:
I don't feel strongly about this I guess. I agree we have existing inconsistency anyhow, so maybe this doesn't matter.